src/client/js/CoreEditor.js

```diff
@@ -166,13 +166,50 @@ class CoreEditor extends React.Component {
                     />
                   </div>
                 </div>
-                <CommentEnabler commentID={this.state.commentID} author={this.props.author} editUrl={this.state.editUrl} gotCommentID={this.gotCommentID.bind(this)}/>
-                <button type="button" className="btn btn-default btn-sm" onClick={() => this.toggleCustomAction('save')}>
-                    Save
-                </button>
-                <button type="button" className="btn btn-default btn-sm" onClick={() => this.toggleCustomAction('saveas')}>
-                    Save as
-                </button>
+                <div className="row ">
+                    <div className="col-xs-6">
+                        <CommentEnabler commentID={this.state.commentID} author={this.props.author} editUrl={this.state.editUrl} gotCommentID={this.gotCommentID.bind(this)}/>
+                    </div>
+                    <div className="col-xs-6">
+                        <button type="button" className="btn btn-default btn-sm pull-right" onClick={() => this.toggleCustomAction('save')}>
+                            Save
+                        </button>
+                        <button type="button" className="btn btn-default btn-sm pull-right" onClick={() => this.toggleCustomAction('saveas')}>
+                            Save as
+                        </button>
+                    </div>
+                </div>
+                <div className="form-group form-inline has-error">
```

Comment:
Make new component called PostSettings in new file, and move theese text it's render() method. This will help us to not over-bloat the code.